azure-container-apps-maven-plugin/src/main/java/com/microsoft/azure/maven/containerapps/AbstractMojoBase.java

```diff
@@ -61,6 +61,10 @@ public abstract class AbstractMojoBase extends AbstractAzureMojo {
     @Parameter(property = "registry")
     protected ContainerRegistryConfig registry;
 
+    @Getter
+    @Parameter(property = "registryIdentity")
```

Comment:
I wonder whether we need to explicitly say `registry` in this configuration. There are two concerns
1. if this is purely registry related setting, it may be better to place `registry` and `registryIdentity` in a group, which results in `{registry: {server: xxx, identity: xxx}}` configuration structure. However, this is likely to be a breaking change
2. identity is an ACA resource concept, and this identity might possibly be used in other scenarios (although it may be true that in Maven plugin context we only cares about AcrPull). Placing `registry` in front may cause confusion.